neural_chat/pipeline/plugins/security/sensitive_checker.py

```diff
@@ -1,8 +1,26 @@
+#!/usr/bin/env python

```

Comment:
which one shall we use? safety_checker or sensitive_checker? I would suggest keyword_filter, as it accurately describes what here does.